mcafee-extension-windows-vm/azuredeploy.json

```diff
@@ -0,0 +1,261 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "metadata": {
+        "description": "Location of resources"
+      }
+    },
+    "newStorageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "Storage Account Name"
+      }
+    },
+    "storageAccountType": {
+      "type": "string",
+      "defaultValue": "Standard_LRS",
+      "metadata": {
+        "description": "Type of the Storage Account"
+      }
+    },
+    "publicIPAddressName": {
```

Comment:
This can become a variable to reduce the number of parameters a user has to enter. Also publicIPAddressType, vmSize, image related parameters, vnet related, subnet related, nicName and extension name can all become variables as well.